### PR TITLE
ICMSLST-445 Create/Edit/Archive/Restore exporter's agents

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

**Review by oskusalerma — APPROVED** [review]
ok after doing the stuff django-doctor suggests

File: web/domains/exporter/models.py [hunk 1]
```diff
@@ -3,12 +3,23 @@
...
 # for efficiency; see https://django-guardian.readthedocs.io/en/stable/userguide/performance.html
 class Exporter(Archivable, models.Model):
+    objects = ExporterManager()
+
     is_active = models.BooleanField(blank=False, null=False, default=True)
-    name = models.CharField(max_length=4000, blank=False, null=False)
-    registered_number = models.CharField(max_length=15, blank=True, null=True)
+    name = models.CharField(
+        max_length=4000, blank=False, null=False, verbose_name="Organisation Name"
+    )
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/exporter/models.py]: fixed in https://github.com/uktrade/icms/pull/204/commits/287a30b91a95b8b2906e2974b4e40479f6f74e97

File: web/domains/importer/models.py [hunk 2]
```diff
@@ -28,11 +28,17 @@ class Importer(Archivable, models.Model):
 
     is_active = models.BooleanField(blank=False, null=False, default=True)
 
-    type = models.CharField(max_length=20, choices=TYPES, blank=False, null=False)
+    type = models.CharField(
+        max_length=20, choices=TYPES, blank=False, null=False, verbose_name="Entity Type"
+    )
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/importer/models.py]: fixed in https://github.com/uktrade/icms/pull/204/commits/287a30b91a95b8b2906e2974b4e40479f6f74e97

File: web/domains/importer/models.py [hunk 3]
```diff
@@ -28,11 +28,13 @@ class Importer(Archivable, models.Model):
 
     is_active = models.BooleanField(blank=False, null=False, default=True)
 
-    type = models.CharField(max_length=20, choices=TYPES, blank=False, null=False)
+    type = models.CharField(max_length=20, choices=TYPES, verbose_name="Entity Type")
 
     # these are only set for organisations
-    name = models.CharField(max_length=4000, blank=True, null=True)
-    registered_number = models.CharField(max_length=15, blank=True, null=True)
+    name = models.TextField(blank=True, null=True, verbose_name="Organisation Name")
```
[the remark this answers is not in the document]
Comment (richtier) [reply web/domains/importer/models.py]: :wave: I'm the chap behind Django Doctor. I see he should have suggested this with his orig suggestion. Will fix that this evening so it's _more helpful_ and _less noisy_. Thanks for bearing with the noise in the meantime

File: web/domains/importer/models.py [hunk 4]
```diff
@@ -28,11 +28,17 @@ class Importer(Archivable, models.Model):
...
+    type = models.CharField(
+        max_length=20, choices=TYPES, blank=False, null=False, verbose_name="Entity Type"
+    )
 
     # these are only set for organisations
-    name = models.CharField(max_length=4000, blank=True, null=True)
-    registered_number = models.CharField(max_length=15, blank=True, null=True)
+    name = models.CharField(
+        max_length=4000, blank=True, null=True, verbose_name="Organisation Name"
+    )
```
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/importer/models.py]: fixed in https://github.com/uktrade/icms/pull/204/commits/287a30b91a95b8b2906e2974b4e40479f6f74e97

File: web/domains/importer/models.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (kevinetienne) [reply web/domains/importer/models.py]: Good catch thanks @richtier. I like the fact that the linter take into account single quotes and double quotes too. But I guess it's harder for the suggestion to guess if the project is using one or the other.

[the remark this answers is not in the document]
Comment (richtier) [reply web/domains/importer/models.py]: :smiley: I have on the roadmap inferring the code style (e.g, if black is used in the repo then run that on the generated code so it's the same style, with a fallback to guessing based on the code that is being replaced for when black is not used

**Pull request merged**